### PR TITLE
Accepted stories can't change.

### PR DESCRIPTION
**Pull request by akitaonrails**
Notes, tasks and attachments should also be frozen.
Adjustments must be new stories.

### CODE REVIEW

File: app/models/story.rb [hunk 1]
```diff
@@ -1,4 +1,11 @@
 class Story < ActiveRecord::Base
+  module ReadOnlyDocuments
+    def documents=(attachments)
```
Comment (adbatista) [inline app/models/story.rb]: don't AR do this by default?

**Comment by adbatista** [comment]
maybe should be good, remove update actions from the view too

**Comment by adbatista** [comment]
:+1:

File: app/models/story.rb (hunk 1 above)
Comment (akitaonrails) [inline app/models/story.rb]: nope, the "documents=" is defined dynamically by `has_attachments :documents` from attachinary. the AR `has_many` can have a `readonly: true` but this is defined in the class level and I didn't find any way to redefine readonly association on the fly. But again, documents is not an association it's a method that links to Cloudinary::Upload underneath. Didn't find any easy way to disallow the setter on a readonly model.

**Pull request merged**